### ISSUE TITLE
d3-scale-chromatic Definitions

### ISSUE THREAD

**Issue by Ledragon** [tomwanzek/d3-v4-definitelytyped#55]
Hi,
Knowing it is not part of the standard bundle, I don't know if it is something of interest for this repo, but I started to use [d3-scale-chromatic](https://github.com/d3/d3-scale-chromatic), which provides quick access to the [color brewer](http://colorbrewer2.org/) color palettes, and thus wrote some (limited amount) of the API.
Would it make sense to complete it and submit a PR in this repo, or should it go directly to DT?

**Comment by tomwanzek** [comment]
I am aware of the module, but have not looked at it since it was created. So I am not sure if it is alpha/beta/RC or stable?

In broad terms I would prioritize as follows:
1. Complete TS 2 versions of definitions cum tests for all standard module bundles and migration to DefinitelyTyped/types-2.0.
2. Convert complete modules to TS 1.8.x and migrate to DefinitelyTyped/master

As with your suggested `d3-scale-chromatic`, I could see putting some effort into `d3-selection-multi`, which is likewise not in the standard bundle but valuable.

As long as the migration is not complete, I can see the value of adding these **extras** under a separate milestone in this repo. The main idea being the ability to compile/test in an integrated fashion and keep consistency.

In short, I think we could add it here and then migrate, but keeping our eyes on the main prize.

For me that means (other than the two you already started:
- d3-force (my todo),
- d3-dsv (open),
- d3-queue (open)

**Comment by Ledragon** [comment]
It is in version 1.0.1, so I guess it is stable.
I will for sure not make it my number one priority, I just wondered if those "satellites" modules would find their place in the repo; I now have an answer to that.
I will keep going on with geo as soon as I have time, my week-end is kinda busy though ;-)

**Comment by tomwanzek** [comment]
I started with the update of the README/Contributing and specifically referred to d3-scale-chromatic and d3-selection-multi in it. Where I would draw an absolute line in the sand for this repo, are definitions for any D3 extension module that are not authored by Mike Bostock as a lead. Even if they are published under the **d3** organization.

**Comment by Ledragon** [comment]
Sound very clear, and a reasonable way to draw the line. Totally on board with this.

**Comment by Ledragon** [comment]
Started [here](https://github.com/Ledragon/d3-v4-definitelytyped/tree/d3-scale-chromatic)

**Comment by tomwanzek** [comment]
Thanks for taking this one on next!

**Comment by tomwanzek** [comment]
@Ledragon just wondering, if you want to pull these definitions still through here for old time's sake or submit them directly as a PR to **DefinitelyTyped/types-2.0** As all the standard modules + the already complete optional scope modules are now merged and available through `@types`, I intend to wind this repo down for purposes of collaboration.

Anything that is still open, I will migrate over to DT i.p. open issues. I will cross-reference and keep this repo for historic reference.

With an updated README of course.

So given all your hard work on this in the past :+1: , I leave it up to you! Let me know.

**Comment by Ledragon** [comment]
I am on the test phase, so it that is OK with you I'd rather submit it through here, as the fork and the branch are already created. I'll do my best to finish it ASAP so that I don't hold you down.

**Comment by tomwanzek** [comment]
Hey, as I said, all up to you. So I will build a nice fence around you :smile:. Whenever you are ready we will pull it through both repos. No worries. Cheers, Tom

**Comment by Ledragon** [comment]
PR submitted :wink:

**Comment by tomwanzek** [comment]
Thanks as always. I will have a look as soon as I can. Just migrating some things over to angular-cli beta.14/Angular 2.0.0... never a dull moment. Then I will also migrate/restructure the open issues on DT and update the README here for ease of reference.

Will cc you as I go along.

**Comment by Ledragon** [comment]
Sure no rush, thanks for the follow-up

**Comment by tomwanzek** [comment]
Thanks a lot. Will migrate over to DT and cc you on the PR. :tada: Closed with #118.
